Tasks/merge.js

```diff
@@ -1,16 +1,13 @@
 // Merge two dictionaries
 
-merge_two_objects = function merge_two_objects(object_1, object_2) {
-  object_3 = [{}][0];
-  for (attribute_name in object_1) {
-    object_3[attribute_name] == object_1[attribute_name];
-    object_3[attribute_name] = object_1[attribute_name];
+const merge = (object1 = {}, object2 = {}) => {
+ const object3 = { ...object1};
+  const keys2 = Object.keys(object2);
+  for (const key of keys2){
```

Comment:
This will not result in a syntax error, but if you do it so, it will be more readable:
```suggestion
  for (const key of keys2) {
```